src/pages/workspace/WorkspaceBankAccountPage.js

```diff
@@ -58,6 +59,14 @@ class WorkspaceBankAccountPage extends React.Component {
         this.navigateToBankAccountRoute();
     }
 
+    componentDidUpdate(prevProps) {
```

Comment:
Hmmm ....

So, you're doing this because you want to wait for the RestartBankAccountSetup call to finish (which deletes the free plan bank account) before calling navigateToBankAccountRoute (which calls fetchFreePlanVerifiedBankAccount), so it doesn't end up fetching a deleted bank account.

Personally, I would prefer calling `API.makeRequestWithSideEffects('RestartBankAccountSetup'` than doing this even though I know we want to avoid it.

Or another alternative would be to somehow pass a param to navigateToBankAccountRoute to tell it not to call fetchFreePlanVerifiedBankAccount in this case, as we should just be able to rely on the optimistic data that we put (that resets REIMBURSEMENT_ACCOUNT)

Reply:
Thats interesting. I wonder if we can do a bit of both. There are actually two problems here:
1. User resets VBBA and then we immediately do a fetch
2. When user logs in for the first time, we don't do any sort of fetch before `WorkspaceBankAccountPage`, so we skip this step entirely.  Which is a bit strange
    ![image](https://user-images.githubusercontent.com/12980144/197900263-60c1203b-ceeb-4894-80be-83af70910809.png)


We could:
1. Always fetch VBBA upon getting to this page
2. If we navigate to `/bank-account` from this component, we _always_ skip fetching (since we just did one)
3. If user chooses to "Start Over", we call the API. Skip the fetch, so we have no issue.

Reply:
Sounds good to me

Reply:
Passing in a param to navigate is trickier than I thought. Still working